dask_sql/physical/rel/logical/aggregate.py

```diff
@@ -327,9 +327,10 @@ def _collect_aggregations(
 
         for expr in rel.aggregate().getNamedAggCalls():
             # Determine the aggregation function to use
-            assert (
-                expr.getExprType() == "AggregateFunction"
-            ), "Do not know how to handle this case!"
+            assert expr.getExprType() in {
+                "AggregateFunction",
+                "AggregateUDF",
+            }, "Do not know how to handle this case!"
 
             # TODO: Generally we need a way to capture the current SQL schema here in case this is a custom aggregation function
             schema_name = "root"
```

Comment:
This is my fault. I am the one who originally made this naughty bit. Figure we can change it now before we forget to however

```suggestion
            schema_name = context.schema_name
```